### PR TITLE
Fix all of the playground type errors

### PR DESCRIPTION
**Pull request by thegreatercurve**
- Fix all of the playground type errors.
- Remove old `$FlowFixMes`.

### CODE REVIEW

File: packages/lexical-playground/src/Editor.tsx [hunk 1]
```diff
@@ -57,6 +57,13 @@ import YouTubePlugin from './plugins/YouTubePlugin';
 import ContentEditable from './ui/ContentEditable';
 import Placeholder from './ui/Placeholder';
 
+interface customWindow extends Window {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  parent: any;
+}
```
Comment (zurfyx) [inline packages/lexical-playground/src/Editor.tsx:63]: Why do we need a custom Window for this?

File: packages/lexical-playground/src/nodes/ExcalidrawNode/ExcalidrawModal.tsx [hunk 2]
```diff
@@ -40,11 +37,11 @@ type Props = {
   /**
    * Handle modal closing
    */
-  onHide: () => mixed;
+  onHide: () => unknown;
```
Comment (zurfyx) [inline packages/lexical-playground/src/nodes/ExcalidrawNode/ExcalidrawModal.tsx:40]: I guess we could just force `void` but that's fine too?

File: packages/lexical-playground/src/nodes/ExcalidrawNode/ExcalidrawModal.tsx [hunk 3]
```diff
@@ -40,11 +37,11 @@ type Props = {
...
   /**
    * Callback when the save button is clicked
    */
-  onSave: (elements: ReadonlyArray<ExcalidrawElementFragment>) => mixed;
+  onSave: (elements: ReadonlyArray<ExcalidrawElementFragment>) => unknown;
```
Comment (zurfyx) [inline packages/lexical-playground/src/nodes/ExcalidrawNode/ExcalidrawModal.tsx:44]: ^

**Review by zurfyx — APPROVED** [review]

File: packages/lexical-playground/src/Editor.tsx (hunk 1 above)
Comment (thegreatercurve) [reply packages/lexical-playground/src/Editor.tsx:63]: `right` is undefined on the `frames`

File: packages/lexical-react/LexicalMarkdownShortcutPlugin.d.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@
  *
  */
 
+import type {Transformer} from '@lexical/markdown';
```
Comment (thegreatercurve) [inline packages/lexical-react/LexicalMarkdownShortcutPlugin.d.ts:9]: +1 for TS @fantactuka. This created a TS error as we were importing the wrong kind of Transformer type.

**Review by zurfyx — APPROVED** [review]

**Pull request merged**